public/components/FeatureAnywhereContextMenu/CreateAnomalyDetector/AddAnomalyDetector.tsx

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import {
+  EuiFlexItem,
+  EuiFlexGroup,
+  EuiFlyoutHeader,
+  EuiFlyoutBody,
+  EuiFlyoutFooter,
+  EuiTitle,
+  EuiSpacer,
+  EuiButton,
+  EuiButtonEmpty,
+  EuiFormFieldset,
+  EuiCheckableCard,
+} from '@elastic/eui';
+import './styles.scss';
+import CreateNew from './CreateNew';
+
+function AddAnomalyDetector({
+  embeddable,
+  closeFlyout,
+  core,
+  services,
+  mode,
+  setMode,
+  index,
+}) {
+  const onCreate = () => {
+    console.log(`Current mode: ${mode}`);
+    const event = new Event('createDetector');
+    document.dispatchEvent(event);
+    closeFlyout();
+  };
+
+  return (
+    <div className="add-anomaly-detector">
+      <EuiFlyoutHeader hasBorder>
+        <EuiTitle>
+          <h2 id="add-anomaly-detector__title">Add anomaly detector</h2>
+        </EuiTitle>
+      </EuiFlyoutHeader>
+      <EuiFlyoutBody>
+        <div className="add-anomaly-detector__scroll">
+          <EuiFormFieldset
+            legend={{
+              display: 'hidden',
+              children: (
+                <EuiTitle>
+                  <span>Options to create a new detector or associate an existing detector</span>
+                </EuiTitle>
+              ),
+            }}
+            className="add-anomaly-detector__modes"
+          >
+            {[
+              {
+                id: 'add-anomaly-detector__create',
+                label: 'Create new detector',
+                value: 'create',
```

Comment:
can we have `create`/`existing` in an enum that can be referenced here? I believe we have something similar for existing AD creation flow but not sure